package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Assists in the creation of ATNA audit trail messages for IHE profiles.",
   "main": "index.js",
   "scripts": {
-    "test": "tap --cov test.js"
+    "pretest": "npm run gen:rng",
+    "test": "tap --cov test.js",
+    "gen:rng": "sed 's/#.*//' dicom.rnc > temp.rnc && trang temp.rnc dicom.rng && rm temp.rnc"
```

Comment:
Oh okay. It'll probably take a bit of work figuring out which ones are causing the issue (... maybe something with the double hashed comments, which I believe has special meaning). So no objections from my side to leave this for now :)